src/Emanote/Model/Stork/Index.hs

```diff
@@ -59,7 +59,7 @@ storkBin = $(staticWhich "stork")
 
 runStork :: MonadIO m => Input -> m LByteString
 runStork input = do
-  let storkToml = handleTomlandBug $ Toml.encode inputCodec input
+  let storkToml = handleTomlandBug $ Toml.encode storkInputCodec $ StorkInput input
```

Comment:
Why this change?

I don't see any reason for adding `storkInputCode` and `StorkInput` either.

Reply:
I tried using only `inputCodec`, using `input.frontmatter_handling` but the generated file looked like

```toml
input.frontmatter_handling = "Ignore"

[[input.files]]
...
```

And Stork didn't like it.

Using another function with `Toml.table` was the only way I got it to work.

I might have missed something, though. I'm quite new to Haskell.

Reply:
That's fair, I'll take a look.